.env

```
@@ -1,15 +1,15 @@
 # Whitelabelling envs
 
-REACT_APP_TITLE=CARE
-REACT_APP_META_DESCRIPTION="Revolutionizing EMR with AI: Open Healthcare Network develops free, open-source tools to enhance efficiency in global healthcare delivery. Our EMR system is recognized as a Digital Public Good by the United Nations."
+REACT_APP_TITLE=HEVAH MEDICAL CLINIC
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)